### PR TITLE
05 Iteration

### PR DESCRIPTION
**Pull request by zentox**
Small enhancements and new features:
- Added an option to add/remove courses as favourite
- Display favourite on the profile of the user
- Hyperlinks to the table of content for the course
- Embedding graphics in LaTeX and an information how to use it as tooltip
- Translations
- Reverting courses and content of a course
- Creating topics in the course structure
- Adding existing topics to the course structure
- Drag & Drop option for topics to reorder topics in the course structure

### CODE REVIEW

File: requirements.txt.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+XXXXXXXXXXXXX
```
Comment (lizz-py) [inline requirements.txt.py:1]: What are these symbols?

**Review by lizz-py — APPROVED** [review]
Maybe delete all the line numbers in translation, since we are updating code in every iteration and the lines are being shifted. Or just update them at the end of the project.

content/templates/content/view/invalid.html: fehlt eine Übersetzung

export/helper_functions.py : pre_render has a typo in type no_error and there is 2-times type no_error instead of one time type and one time parameter

Help-Text of Latex Image Attachments could be more detailled

HTML change " to ' in all files (mostly for icons or trans)

Remove the TODO comments in some files

File: requirements.txt.py (hunk 1 above)
Comment (zentox) [reply requirements.txt.py:1]: This file should not have exited and will be removed.

**Comment by zentox** [comment]
> Maybe delete all the line numbers in translation, since we are updating code in every iteration and the lines are being shifted. Or just update them at the end of the project.

Will be discussed in the next iteration how that will be resolved.

> content/templates/content/view/invalid.html: fehlt eine Übersetzung

Fixed.  62ce7b2

> export/helper_functions.py : pre_render has a typo in type no_error and there is 2-times type no_error instead of one time type and one time parameter

Fixed.  62ce7b2

> Help-Text of Latex Image Attachments could be more detailled

Will be resolved in the next iteration.

> HTML change " to ' in all files (mostly for icons or trans)

Fixed.  62ce7b2

> Remove the TODO comments in some files

Fixed.  62ce7b2

**Pull request merged**